sphinxgallery/gen_rst.py

```diff
@@ -225,7 +225,7 @@ def _plots_are_current(src_file, image_file):
     return not needs_replot
 
 
-def save_figures(image_path, fig_count):
+def save_figures(image_path, fig_count, gallery_conf):
```

Comment:
Not in this case. There is a copy of the config in  app.config.sphinxgallery_conf if the function is directly called by sphinx.